docker-compose.yml

```diff
@@ -343,6 +363,28 @@ services:
       KAFKA_LDAP_USER_NAME_ATTRIBUTE: uid
       KAFKA_LDAP_USER_OBJECT_CLASS: inetOrgPerson
 
+      # ConfluentTelemetryReporter: Disable
+      KAFKA_CONFLUENT_TELEMETRY_REPORTER_ENABLE: 'false'
```

Comment:
@rnpridgeon Looks like this one was missed for deletion?  (see https://github.com/confluentinc/cp-demo/pull/239/commits/675ae669d75814c9fe4e829c268ec77f2d8c0486 for deletion on other broker config)